react/components/RadioGroup/index.js

```diff
@@ -19,9 +19,12 @@ class RadioGroup extends React.Component {
       label,
       testId,
       size,
+      error,
+      errorMessage,
     } = this.props
 
     const large = size === 'large'
+    const borderColor = (error || errorMessage) ? 'b--danger hover-b--danger' : 'b--muted-4'
```

Comment:
I had doubts about this condition, but our `Input` works in the [same way](https://github.com/vtex/styleguide/blob/f67a7e2967db7f16b2476ee68c2871712b699f2b/react/components/Input/index.js#L138).

Reply:
Yeah, me too. I have used the `Input` as an example to keep consistency.